python/mp/rosary.py

```diff
@@ -161,6 +161,21 @@ def __init__(self,
             'tungsten100': color.Color(1, 214/255, 170/255),
             'tungsten40': color.Color(1, 197/255, 143/255),
             'candle': color.Color(1, 147/255, 41/255),
+            'fire': color.ColorMapRandomWalk(
+                colormap=color.ColorMap(colormap=[
+                    color.ColorMapStep(step=0, color=color.Color(1, 132/255, 41/255)),
+                    color.ColorMapStep(step=1/6, color=color.Color(1, 137/255, 41/255)),
+                    color.ColorMapStep(step=2/6, color=color.Color(1, 142/255, 41/255)),
+                    # The middle value here is taken directly from the
+                    # "Reproducing Real World Light" article:
+                    # http://planetpixelemporium.com/tutorialpages/light.html
+                    color.ColorMapStep(step=3/6, color=color.Color(1, 147/255, 41/255)),
+                    color.ColorMapStep(step=4/6, color=color.Color(1, 152/255, 41/255)),
+                    color.ColorMapStep(step=5/6, color=color.Color(1, 157/255, 41/255)),
+                    color.ColorMapStep(step=1, color=color.Color(1, 162/255, 41/255))
+                ]),
+                time=3
+            ),
```

Comment:
I really like that you added the new color here, so it just has a name like any other color. I don't know what it didn't occur to me to do this with ColorMapFade() before.